blacksauce/src/_includes/_footer.html

```diff
@@ -0,0 +1,33 @@
+<footer class="u-black footer_padding">
+	<h2 class="u-brandOrange u-marginNone h2--callToAction">Get In Touch</h2>
+	<p class="u-white p--contactInfo">Contact us for any questions about catering, the events we’re at, biscuits in
+		general or just to
+		say hello. Use the form or contact us directly via <a class="a__email" href="mailto:info@blacksaucekitchen.com">info@blacksaucekitchen.com</a>.
+		We’re based in Baltimore,
+		Maryland, but you can find us at events in
+		a wide range around our home base.</p>
+	<form action="">
+		<div class="formBoxGrid">
+			<label for="name">Name*</label>
+			<input class="formBoxPosition" type="text" id="name" name="name" value="" required>
+		</div>
+		<div class="formBoxGrid">
+			<label for="email">Email*</label>
+			<input class="formBoxPosition" type="email" id="email" name="email" value="" required>
+		</div>
+		<label for="message">Message*</label>
+		<textarea id="styled" id="message" name="message" required></textarea>
```

Comment:
This element has 2 IDs; did you mean for one to be a class:

```suggestion
 		<textarea class="styled" id="message" name="message" required></textarea>
```